tests/fixture_plugins/quality_controller_fixtures/sequencing_qc_fixtures.py

```diff
@@ -0,0 +1,150 @@
+def express_sample_fail_sequencing_qc(
+    sequencing_qc_check_scenarios: SequencingQCCheckScenarios,
+) -> Sample:
+    return sequencing_qc_check_scenarios.sample_scenario(
+        priority=Priority.express,
+        prep_category=PrepCategory.WHOLE_GENOME_SEQUENCING,
+        pass_sequencing_qc=False,
+    )
+
+
+@pytest.fixture(scope="function")
+def sample_pass_sequencing_qc(
+    sequencing_qc_check_scenarios: SequencingQCCheckScenarios,
+) -> Sample:
+    return sequencing_qc_check_scenarios.sample_scenario(
+        priority=Priority.standard,
+        prep_category=PrepCategory.WHOLE_GENOME_SEQUENCING,
+        pass_sequencing_qc=True,
+    )
+
+
+@pytest.fixture(scope="function")
+def sample_fail_sequencing_qc(
+    sequencing_qc_check_scenarios: SequencingQCCheckScenarios,
+) -> Sample:
+    return sequencing_qc_check_scenarios.sample_scenario(
+        priority=Priority.standard,
+        prep_category=PrepCategory.WHOLE_GENOME_SEQUENCING,
+        pass_sequencing_qc=False,
+    )
+
+
+@pytest.fixture(scope="function")
+def case_pass_sequencing_qc(
+    sequencing_qc_check_scenarios: SequencingQCCheckScenarios,
+) -> Sample:
+    return sequencing_qc_check_scenarios.case_scenario(
+        priority=Priority.standard,
+        pass_sequencing_qc=True,
+        prep_category=PrepCategory.WHOLE_GENOME_SEQUENCING,
+        workflow=Workflow.MIP_DNA,
+    )
+
+
+@pytest.fixture(scope="function")
+def case_fail_sequencing_qc(
+    sequencing_qc_check_scenarios: SequencingQCCheckScenarios,
+) -> Sample:
+    return sequencing_qc_check_scenarios.case_scenario(
+        priority=Priority.standard,
+        pass_sequencing_qc=False,
+        prep_category=PrepCategory.WHOLE_GENOME_SEQUENCING,
+        workflow=Workflow.MIP_DNA,
+    )
+
+
+@pytest.fixture(scope="function")
+def express_case_pass_sequencing_qc(
+    sequencing_qc_check_scenarios: SequencingQCCheckScenarios,
+) -> Sample:
+    return sequencing_qc_check_scenarios.case_scenario(
+        priority=Priority.express,
+        pass_sequencing_qc=True,
+        prep_category=PrepCategory.WHOLE_GENOME_SEQUENCING,
+        workflow=Workflow.MIP_DNA,
+    )
+
+
+@pytest.fixture(scope="function")
+def express_case_fail_sequencing_qc(
+    sequencing_qc_check_scenarios: SequencingQCCheckScenarios,
+) -> Sample:
+    return sequencing_qc_check_scenarios.case_scenario(
+        priority=Priority.express,
+        pass_sequencing_qc=False,
+        prep_category=PrepCategory.WHOLE_GENOME_SEQUENCING,
+        workflow=Workflow.MIP_DNA,
+    )
+
+@pytest.fixture(scope="function")
+def any_sample_in_case_has_reads(
+    sequencing_qc_check_scenarios: SequencingQCCheckScenarios,
+) -> Sample:
+    return sequencing_qc_check_scenarios.case_scenario(
+        priority=Priority.standard,
+        pass_sequencing_qc=True,
+        prep_category=PrepCategory.WHOLE_GENOME_SEQUENCING,
+        workflow=Workflow.MIP_DNA,
+    )
+    
+@pytest.fixture(scope="function")
+def any_sample_in_case_no_reads(
```

Comment:
This is unclear, maybe
```suggestion
def case_with_samples_with_no_reads(
```